### DISCUSSION TITLE
Volunteer as maintainer

### DISCUSSION THREAD

**Discussion by eldruin** [serialport/serialport-rs#83] [General]
Hi, I do not have a whole lot of time but I think I could help a bit here if you are interested.
We are in the process of replacing serial-rs with this library over at https://github.com/rust-embedded/linux-embedded-hal/pull/89.

**Comment by jessebraham** [comment]
That would be fantastic, thank you for volunteering! Any help here would be great, unfortunately due to my work I just don't have the time or energy to focus on this project right now (and likely won't for the foreseeable future).

I will get in touch with the owner of the organization and get you added to the team, thanks again!

**Comment by jessebraham** [comment]
You should have received an invite, please let me know if there are any issues!

You can find me in our Matrix channel if you need anything: https://matrix.to/#/#serialport-rs:matrix.org

**Comment by eldruin** [comment]
Thanks! I just joined the org and the matrix channel :)
